src/block/Cobweb.php

```diff
@@ -39,6 +39,9 @@ public function onEntityInside(Entity $entity) : bool{
 	}
 
 	public function getDropsForCompatibleTool(Item $item) : array{
+		if($item->getBlockToolType() === BlockToolType::SHEARS){
```

Comment:
```suggestion
		if(($item->getBlockToolType() & BlockToolType::SHEARS) !== 0){
```
Items may be multiple block tool types at the same time (e.g. plugins may implement supertools which can break any block).